.github/workflows/release-dry-run.yml

```diff
@@ -40,4 +40,4 @@ jobs:
           bundle install --jobs 4 --retry 3
       - name: Build
         run: |
-          ./fourier bundle all --xcode_version=$(cat .xcode-version) --xcode_version_libraries=$(cat .xcode-version-libraries)
+          ARGS="--xcode-version $(cat .xcode-version) --libraries-xcode-version $(cat .xcode-version-libraries) --build-dir ./build" make release/bundle
```

Comment:
Why do we need to pass these from external? Can't the script read it on its own?

Reply:
I thought it might be helpful to have flexibility over that when invoking the script to test it against multiple versions of Xcode.

Reply:
I applied your suggestion here.